features/step_definitions/delete.ts

```diff
@@ -13,3 +13,21 @@ Then(
     assert.equal(recordNumbers.length, numberOfRecords);
   },
 );
+
+When("I run the delete command with args {string}", function (args: string) {
+  this.execCliKintone(args);
+});
+
+When("I type {string}", function (userInput) {
+  this.childProcess.stdin.write(userInput);
+});
+
+When("I press Enter", function () {
+  return new Promise<void>((resolve) => {
+    this.childProcess.stdin.write("\n");
+    this.childProcess.stdin.end();
+    this.childProcess.on("close", () => {
+      resolve();
+    });
+  });
+});
```

Comment:
These steps can be in the `common.ts` because it doesn't have domain features that are only for the record delete command.

Reply:
@mshrtsr 

Currently, the step "I run the delete command with args {string}" contains the domain feature (`delete`).
So, I keep it at `delete.ts`.

If not, we need to change to step definition, such as:

`I run the prompt command with args {string}`

Or, do you have any suggestions?

Reply:
@hung-cybo 
The difference between `execCliKintone` and `execCliKintoneSync` is just whether cli-kintone is interactive(prompt) or not.
So I think it's better to move it to `common.ts` by renaming the step definition.

> I run the prompt command with args {string}
It seems good!
For me, `I run the command with a prompt with args` is more sounds natural, but I leave it to you due to a small difference.